src/components/login/login.scss

```diff
@@ -28,22 +28,22 @@
         font-weight: bold;
     }
 
-    .right {
-        float: right;
-    }
-
     .spinner {
         margin: 0 .8rem;
         width: 1rem;
         vertical-align: middle;
     }
+    
+    .submit-row {
```

Comment:
I tested this, and it works great except that at widths under 768px, the `flex-direction` of `FlexRow` components switches to `column`; so to prevent the button from being centered in an ugly way, we need to enforce `row` here, like:

```
    .submit-row {
        justify-content: space-between;
        flex-direction: row; // override centered columns
    }
```

Reply:
Good catch - I'd forgotten about the switch to centered column. I think the `FlexRow` component is poorly named. Another bit of tech debt.